ethereum/stratum/src/main/java/org/hyperledger/besu/ethereum/stratum/StratumServer.java

```diff
@@ -89,11 +114,18 @@ public StratumServer(
   }
 
   private void handle(final NetSocket socket) {
+    connectionsCount.inc();
+    numberOfMiners.incrementAndGet();
```

Comment:
It's interesting the atomic APIs don't have a simple `imcrement` or `add` method, they all require returning a value. but `set` works just fine. </rant>

Reply:
Indeed, I was looking for it. It's ok, not a big deal.